django/contrib/admin/static/admin/css/responsive.css

```diff
@@ -0,0 +1,984 @@
+        width: auto;
+    }
+
+    .inline-group {
+        overflow: auto;
+    }
+
+    input[type="submit"], button {
+        -webkit-appearance: none;
+        appearance: none;
+    }
+
+    .aligned .form-row {
+        max-width: calc(100vw - 100px);
+        overflow: auto;
+    }
+
+    .selector {
+        display: flex;
+        width: 100%;
+    }
+
+    .selector select {
+        width: 100%;
+    }
+
+    .selector .selector-filter {
+        display: flex;
+        align-items: center;
+    }
+
+    .selector .selector-filter label {
+        margin: 0;
+    }
+
+    .selector-available, .selector-chosen {
+        width: auto;
+        flex: 1 1;
+    }
+
+    .selector ul.selector-chooser {
+        align-self: flex-start;
+    }
+
+    .selector .selector-available input {
+        width: auto;
+        flex: 1 1;
+    }
+
+    .stacked {
+        flex-wrap: wrap;
+        max-width: 490px;
+    }
+
+    .stacked > * {
+        flex: 0 0 100%;
+    }
+
+    .stacked .selector-chooser {
+        flex-basis: 50px;
+        margin-left: auto !important;
+        margin-right: auto !important;
+    }
+
+    #changelist-search div {
+        display: flex;
+        flex-wrap: wrap;
+    }
+
+    #changelist-search div > * {
+        flex: 0 1 auto;
+    }
+
+    #changelist-search .quiet {
+        flex: 1 1 100%;
+        margin: 4px 0 0 24px;
+    }
+
+    #changelist #toolbar form #searchbar {
+        flex: 1 0;
+        margin: 0 10px 0 5px;
+    }
+
+    #changelist #toolbar form input[type="submit"] {
+        flex-shrink: 0;
+    }
+
+    #changelist .actions {
+        display: flex;
+        flex-wrap: wrap;
+        padding: 10px 0;
+        margin-bottom: 10px;
+    }
+
+    #changelist .actions.selected {
+        border: none;
+    }
+
+    #changelist .actions label {
+        display: flex;
+        flex: 1 1;
+        padding-right: 10px;
+    }
+
+    #changelist .actions select {
+        width: 100%;
+        margin-left: 15px;
+    }
+
+    #changelist .actions .button {
+        min-width: 48px;
+    }
+
+    #changelist .actions span.action-counter {
+        flex: 1 0 100%;
+        margin: 0;
+        font-size: 11px;
+    }
+
+    #changelist .actions span.all,
+    #changelist .actions span.question,
+    #changelist .actions span.clear {
+        flex: 1 0 100%;
+        font-size: 11px;
+        margin-left: 0;
+    }
+
+    #changelist-filter {
+        width: 200px;
+    }
+
+    .change-list .filtered .results,
+    .change-list .filtered .paginator,
+    .filtered .actions,
+    .filtered #toolbar,
+    .filtered div.xfull {
+        margin-right: 240px;
+    }
+
+    div.olMap[id] {
+        margin-bottom: 10px !important;
+    }
+
+    pre.literal-block {
+        padding: 10px;
+        overflow: auto;
+    }
+
+    table.model tbody th,
+    table.model tbody td {
+        font-size: 13px;
+        word-break: break-word;
+    }
+
+    table.model tbody th {
+        width: auto;
+    }
+
```

Comment:
I would go with 1 blank line everywhere (rather than two) -- use a comment if a section heading is needed